README.md

```diff
@@ -24,22 +24,14 @@ Import yeldaChat lib and yeldaWebchatInjector.min.css
   import 'yelda-webchat/dist/css/yeldaWebchatInjector.min.css'
 ```
 
-## Load js and css files using script and link tags
-Add the style using the `<link>` tag in the webpage `<head>` tag
-
-```html
-  <link rel="stylesheet" type="text/css" href="/node_modules/yelda/dist/css/yeldaWebchatInjector.min.css"/>
-```
-
```

Comment:
this one need to be kept : you can't be sure that external developer kept the code at the same place